Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerRepositoryImpl.kt

```diff
@@ -0,0 +1,121 @@
+package com.example.jetcaster.ui.player
+
+import com.google.android.horologist.annotations.ExperimentalHorologistApi
+import com.google.android.horologist.media.model.Command
+import com.google.android.horologist.media.model.Media
+import com.google.android.horologist.media.model.PlaybackStateEvent
+import com.google.android.horologist.media.repository.PlayerRepository
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlin.time.Duration
+
+@OptIn(ExperimentalHorologistApi::class)
+class PlayerRepositoryImpl : PlayerRepository {
```

Comment:
Adding a note here that this should use the repository provided in the `core` module. No action needed if you plan on doing that in a separate PR

Reply:
This is the layer in Horologist that allows to customize the Repository, so I need to implement a PlayerRepository but exactly here is where I can decouple from the rest of the implementation and use the Jetcaster one.
By the wat, I couldn't find a repository for the Player but only for the podcasts, is there one? We can work on this together next week.

Reply:
this has been removed